mysql-test/suite/mariabackup/slave_provision_nolock.test

```diff
@@ -97,7 +97,7 @@ send CALL gen_load();
 
 --connection default
 --echo *** Doing backup...
---exec $XTRABACKUP --defaults-file=$MYSQLTEST_VARDIR/my.cnf --backup --target-dir=$basedir --no-lock
+--exec $XTRABACKUP --defaults-file=$MYSQLTEST_VARDIR/my.cnf --backup --target-dir=$basedir --no-lock --innodb_log_checkpoint_now=0
 --echo *** Doing prepare...
```

Comment:
Instead of doing this, what if we executed `SET debug_dbug='+d,…'` on the server and added a `DBUG_EXECUTE_IF` to `checkpoint_now_set()` to allow the statement to be ignored? I think that we’d need that for other reasons as well.

I see that the test does not currently make use of `have_debug.inc`, so it might not be a good idea to make it depend on `CMAKE_BUILD_TYPE=Debug` specific instrumentation.